tensorflow/core/util/mkl_threadpool.h

```diff
@@ -63,12 +63,12 @@ inline void balance211(T n, U team, U tid, T* n_start, T* n_end) {
 struct MklDnnThreadPool : public threadpool_iface {
   MklDnnThreadPool() = default;
 
-  MklDnnThreadPool(OpKernelContext* ctx)
-      : eigen_interface_(ctx->device()
-                             ->tensorflow_cpu_worker_threads()
-                             ->workers->AsEigenThreadPool()) {}
+  MklDnnThreadPool(OpKernelContext* ctx, int num_threads = -1) {
+    eigen_interface_ = ctx->device()->tensorflow_cpu_worker_threads()->workers->AsEigenThreadPool();
+    num_threads_ = (num_threads == -1) ? eigen_interface_->NumThreads(): num_threads;
```

Comment:
Please reformat the source.

Reply:
@penpornk Addressed your comments. Please have a look.